File: app/style.py
# Shared visual constants used across pages.

# FDR colour scheme: maps fixture difficulty rating (1-5) to (background, foreground).
FDR_COLOURS: dict[int, tuple[str, str]] = {
    1: ("darkgreen", "white"),
    2: ("#09fc7b", "black"),
    3: ("#e7e7e8", "black"),
    4: ("#ff1651", "white"),
    5: ("#80072d", "white"),
}

File: web/src/lib/style.ts
// Shared visual constants used across pages.

// FDR colour scheme: maps fixture difficulty rating (1-5) to (background, foreground).
export const FDR_COLOURS: Record<number, { bg: string; fg: string }> = {
	1: { bg: 'darkgreen', fg: 'white' },
	2: { bg: '#09fc7b', fg: 'black' },
	3: { bg: '#e7e7e8', fg: 'black' },
	4: { bg: '#ff1651', fg: 'white' },
	5: { bg: '#80072d', fg: 'white' }
};
